Plotter/python/plot/Plot.py

```diff
@@ -22,7 +22,8 @@
 
 # CMS style
 #TGaxis.SetExponentOffset(-0.074,0.015,'y')
-CMSStyle.extraText  = "Preliminary"
+CMSStyle.extraText  = ""
```

Comment:
It's not crucial for this PR, but why was this set to an empty string? Maybe we should set the default to "Internal", "Private work", "Work in progress", since "Preliminary" is more meant for public plots in PASes.